### PR TITLE
1st Attempt: To-Do Items

### PR DESCRIPTION
**Pull request by tcburns24**
**Checkpoint**: 5
**Description**: Created a model & controller, attempted to write & render a partial, attempted to write tests for items & users. Creating the model, controller, and routes was fairly simple, writing the tests was hard (and is probably incomplete), writing the partial was a fiasco. That's where I'll need guidance. 
**Test Coverage**: 86.96%

### CODE REVIEW

File: spec/factories/items.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  pw = "password"
```
Comment (brittanyjspringer) [inline spec/factories/items.rb]: Tom, please take this out.

File: spec/controllers/items_controller_spec.rb [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
+  before(:each) do
+    @my_user = FactoryGirl.create(:user)
+    sign_in @my_user
+  end
 
   describe "POST create" do
-    it "returns http redirect" do
-      post :create
-      expect(response).to redirect_to(new_session_path)
+    it "increases the number of Post by 1" do
```
Comment (brittanyjspringer) [inline spec/controllers/items_controller_spec.rb]: Should be Item, not Post.

**Comment by tcburns24** [comment]
Hi Brittany,

Made the above changes to Checkpoint 5, committed them into branch **5-to-do-items**, Test Coverage is **86.78%**.
@wonderwoman13

**Comment by tcburns24** [comment]
Checkpoint: 5
Description: Made several attempts at formatting items into a table -- didn't come out as beautifully formatted as I'd  hoped, but this is commit has the most organized result. 
Test Coverage: 86.78%

**Pull request merged**